docs/index.asciidoc

```diff
@@ -47,7 +47,11 @@ NOTE: Some multi-pipeline configurations such as logstash-to-logstash over http(
 
 Events that _fail_ ingest pipeline processing will be tagged with `_ingest_pipeline_failure`, and their `[@metadata][_ingest_pipeline_failure]` will be populated with details as a key/value map.
 
-IMPORTANT: This plugin requires minimum Java 17 and Logstash 8.7.0 versions.
+[id="plugins-{type}s-{plugin}-prerequisites"]
+===== Prerequisites
+- This plugin embeds portions of Elasticsearch Ingest Node whose version matches the `major`.`minor` version of the plugin.
+- When upgrading the {stack} product(s), this plugin should be upgraded _before_ {kib} to ensure that {agent}-managed pipelines have access to the expected functionality.
+- This plugin requires minimum Java 17 with its `8.x` versions and Java 21 with its `9.x` versions.
```

Comment:
```suggestion
[id="plugins-{type}s-{plugin}-requirements"]
===== Requirements and upgrade guidance

- This plugin requires Java 17 minimum with {ls} `8.x` versions and Java 21 minimum with {ls} `9.x` versions.
- When you upgrade the {stack}, upgrade {ls} (or this plugin specifically) _before_ you upgrade {kib}.
  (Note that this requirement is a departure from the typical {stack} https://www.elastic.co/guide/en/elastic-stack/current/installing-elastic-stack.html#install-order-elastic-stack[installation order].) 
+
The {es}-{ls}-{kib} installation order ensures the best experience with {agent}-managed pipelines, and embeds functionality from a version of {es} Ingest Node that is compatible with the plugin version (`major`.`minor`).   
```

Reply:
Renders like this: 
<img width="923" alt="Screenshot 2025-01-07 at 7 51 08 PM" src="https://github.com/user-attachments/assets/dbf74f9c-b4cc-4021-b678-3f7d5fc2e5c6" />

Please free to tweak as you see fit... as long as you don't break anything.  :-D